src/WorkItems/AzureDevOpsFilingClient.cs

```diff
@@ -102,6 +102,12 @@ public override async Task<IEnumerable<WorkItemModel>> FileWorkItems(IEnumerable
                         Operation = Operation.Add,
                         Path = $"/fields/{AzureDevOpsFieldNames.Tags}",
                         Value = string.Join(",", workItemModel.LabelsOrTags)
+                    },
+                    new JsonPatchOperation
+                    {
+                        Operation = Operation.Add,
+                        Path = $"/fields/{AzureDevOpsFieldNames.ReproSteps}",
```

Comment:
>ReproSteps [](start = 64, length = 10)

What will happen here if Repro steps isn't enabled in the target account?

Reply:
I believe it's still added to the work item. i.e. AFAIK, it will still save successfully.

---
In reply to: [398651812](https://github.com/microsoft/sarif-sdk/pull/1830#discussion_r398651812) [](ancestors = 398651812)